ipa-core/src/ff/boolean.rs

```diff
@@ -190,6 +194,18 @@ impl FromRandomU128 for Boolean {
     }
 }
 
+impl AsRef<u8> for Boolean {
```

Comment:
are all the changes for `Boolean` to be able to do this? There may be an easier way to do this

```rust
impl AsRef<u8> for Boolean {
    fn as_ref(&self) -> &u8 {
        match self.0 {
            true => &1,
            false => &0
        }
    }
}
```

Reply:
Yes since I need a bitslice and I can use BitSlice::from_element(u8) but BitSlice::from_element(bool) is not supported by the bitvec crate

Reply:
I'd just change the AsRef implementation in this case then